setup.py

```diff
@@ -0,0 +1,20 @@
+import setuptools
```

Comment:
Nice. It's probably better to start with a `setup.cfg` for new packages though, because using setup.py is deprecated.  You can mostly just translate the lines in the setup.py to analogous lines in setup.cfg. E.g. https://github.com/grahamgower/demesdraw/blob/main/setup.cfg
You'll also need a pyproject.toml file. Some of the details of how things all fit together can be confusing, but there are lots of examples in other python packages.